uqcsbot/base.py

```diff
@@ -198,10 +171,11 @@ def _run_handlers(self, event):
         if "type" not in event:
```

Comment:
(repeating from above):

what's the difference between this `handle_command` and `run_handlers`? docs for both would  be nice